src/components/two-up-display.tsx

```diff
@@ -36,6 +37,7 @@ export class TwoUpDisplayComponent extends BaseComponent<IProps, IState> {
         <div className="content" data-test="two-up-content">
           {this.props.leftPanel}
         </div>
+        <ZoomControl />
```

Comment:
@dstrawberrygirl I just realized that this adds zoom controls to all 2-up displays, including the population view. Depending on the window size, this can cover the populations control.

Would it make sense to add it to the cell-zoom space instead?

Reply:
@sfentress Good call - I have a branch for basic zoom control implementation, so I'll make the change in there.